modules/mono/editor/Godot.NET.Sdk/Godot.SourceGenerators/Common.cs

```diff
@@ -380,19 +398,21 @@ ISymbol exportedMemberSymbol
                     category: "Usage",
```

Comment:
I feel like `GD0303` shouldn't be a diagnostic, since it's not an user error. It definitely shouldn't be _Usage_ since it's not caused by incorrect usage of API, it's a developer error caused by incorrect implementation of the analyzer.